performance/src/resolution/res_plot_tool.cpp

```diff
@@ -8,19 +8,34 @@
 // Library include(s).
 #include "res_plot_tool.hpp"
 
+// Detray include(s).
+#include "detray/utils/statistics.hpp"
+
+// ROOT include(s).
+#ifdef TRACCC_HAVE_ROOT
+#include <TF1.h>
+#include <TGraphErrors.h>
+#endif  // TRACCC_HAVE_ROOT
+
+#include <iostream>
+
 namespace traccc {
 
 res_plot_tool::res_plot_tool(const res_plot_tool_config& cfg) : m_cfg(cfg) {}
 
 void res_plot_tool::book(res_plot_cache& cache) const {
 
     plot_helpers::binning b_pull = m_cfg.var_binning.at("pull");
+    plot_helpers::binning b_eta = m_cfg.var_binning.at("Eta");
+    plot_helpers::binning b_pT = m_cfg.var_binning.at("Pt");
 
     // Avoid unused variable warnings when building the code without ROOT.
     (void)cache;
     (void)b_pull;
+    (void)b_eta;
+    (void)b_pT;
```

Comment:
Btw, does ```[[maybe_unused]]``` not work?

Reply:
It must work. I just worry that it would not work with old compilers (<c++17)

Reply:
Isn't c++17 a minimal requirement in our projects?
https://github.com/acts-project/traccc/blob/d3252a63a87de2817b2e178daaa4fee08b85d028/CMakeLists.txt#L11-L16
Anyway, not that important, I guess

Reply:
Soon it will be C++20 :sunglasses: